### ISSUE TITLE
Distribute source code

### ISSUE THREAD

**Issue by ethan-sargent** [forcedotcom/lwc-dev-server-feedback#161]
**Is your feature request related to a problem? Please describe.**
The lwc dev server is currently unmaintained and cannot easily be improved without source code being made available.

**Describe the solution you'd like**
Source code being made available so that the community can contribute to it and/or maintain it.

**Describe alternatives you've considered**
Rebuilding the local dev server features from scratch - impractical, and throws away the hard work that has already been done to deliver the current state of the local dev server.

**Additional context**
The local dev server is an awesome initiative. Further improvements are currently not possible, and it would be fantastic to see this become something used widely.

**Comment by h0h0h0** [comment]
Yes please +1. At least someone should be looking at tthe repo and updating it!
